packages/front-end/pages/settings/index.tsx

```diff
@@ -16,6 +15,11 @@ import {
   DEFAULT_SEQUENTIAL_TESTING_TUNING_PARAMETER,
   DEFAULT_STATS_ENGINE,
 } from "shared/constants";
+import {
+  OrganizationSettings,
+  SupportedCurrencies,
+} from "@/../back-end/types/organization";
+import { supportedCurrencies } from "@/../back-end/src/util/currencies";
```

Comment:
I don't think you can't import code from the back-end package like this.  You can only import types.  I think this may be working by accident here since that util file is technically just javascript and doesn't contain any typescript syntax.  We added a new `shared` package recently to fix this restriction.  See my other comment about a potentially easier solution of just defining this on the front-end

Reply:
Updated.